group_project/group_project.py

```diff
@@ -244,6 +246,10 @@ def student_view(self, context):
             "ta_graded": (self.group_reviews_required_count < 1),
         }
 
+        user_prefs = self.project_api.get_user_preferences(self.user_id)
+        for activity_component in group_activity.activity_components:
+            activity_component.grading_override = True if "TA_REVIEW_WORKGROUP" in user_prefs else False
```

Comment:
Yes, those are the only actual changes.

When accessing GP view the normal way (by clicking on group work icon on apros side), he should see the normal GP view.

TA grading view should be provided in the following case:
Login to the mckinsey academy site using the credentials from description. 
1. Go to the MCKA ADMIN tab. 
2. Go to the GroupWork. 
3. Select a program from the drop down menu of existing programs. 
4. Select a course from the drop down menu of existing courses. 
5. Click on OK button. 
6. By clicking on OK button it takes the user to Create and Edit Groups Screen. 
7. TA can select any group to grade.(Even if he belongs to a Group) 
8. Select a group